text/0015-market-design-revisit.md

```diff
@@ -0,0 +1,116 @@
+| **Authors**     | Jonas Gehrlein                                                                              |
+
+## Summary
+
+This document is a proposal for restructuring the bulk markets in the Polkadot UC's coretime allocation system to improve efficiency and fairness. The proposal suggests separating the `BULK_PERIOD` into `MARKET_PERIOD` and `RENEWAL_PERIOD`, allowing for a market-driven price discovery through a clearing price Dutch auction during the `MARKET_PERIOD` followed by renewal offers at the `MARKET_PRICE` during the `RENEWAL_PERIOD`. The new system ensures synchronicity between renewal and market prices, fairness among all current tenants, and efficient price discovery, while preserving price caps to provide security for current tenants. It also discusses the initiation of the instantanous market with a discount and the possibility of long-term leases.
+
+## Motivation
+
+While the initial [RFC-1](https://github.com/polkadot-fellows/RFCs/blob/6f29561a4747bbfd95307ce75cd949dfff359e39/text/0001-agile-coretime.md) has provided a robust framework for Coretime allocation within the Polkadot UC, this proposal builds upon its strengths and uses many provided building blocks to address some areas that could be further improved. 
+
+In particular, this proposal introduces the following changes:
+- It introduces a `RESERVE_PRICE` that anchors all markets, promoting price synchronicity among it's substitute goods (flexible/renewed/instantanous coretime). 
+    - This reduces complexity.
+    - This makes sure all tenants pay the same for coretime.
+- It reverses the order of the market and renewal phase. 
+    - This allows to fine-tune the price through market forces.
+- It exposes the renewal prices, while still being beneficial for longterm tenants, more to market forces. 
+- It removes the LeadIn period and introduces a (from the perspective of the coretime systemchain) passive Settlement Phase, that allows the secondary market to exert it's force.
+
+The premise of this proposal is to reduce complexity by introducing a common price (that develops releative to capacity consumption of Polkadot UC), while still allowing for market forces to add efficiency. Longterm lease owners still receive priority **IF** they can pay (close to) the market price. This prevents a situation where the renewal price significantly diverges from renewal prices which allows for core captures. While maximum price increase certainty might seem contradictory to efficient price discovery, the proposed model aims to balance these elements, utilizing market forces to determine the price and allocate cores effectively within certain bounds. It must be stated, that potential price increases remain predictable (in the worst-case) but could be higher than in the originally proposed design. The argument remains, however, that we need to allow market forces to affect all prices for an efficient Coretime pricing and allocation.
```

Comment:
This is somewhat idealistic. Enshrined renewals are "messy" from a purist perspective, and I would also like to see them go away eventually, but in practice they're a better offering until market infrastructure for renewals matures significantly (so longer-term stable products can be built on top of this mechanism). This has two key parts:
  * coretime hedging products
  * technology stacks for automated smart-contract bidders in the auction

I would much prefer revisiting a proposal removing enshrined renewals when the coretime market is more mature.

Reply:
I tried to mitigate the risk of having to change the system later by making it more robust to potential different market situations. In particular, I'd be afraid of core capturing by giving price guarantees to long-term projects, that have a strong interest not to change the system later and give away their privileges. By that time, they might have a large user base and strong influence on governance. 

In times of low demand or low competition (i.e., more supply than demand) for coretime, we'd expect little to no bidding in the `MARKET_PERIOD`. That would lead to a reversal to the `RESERVE_PRICE`. Then, the system boils down to moving the price up or down, equal to RFC-1, *only* by making adjustments through capacity (with `BULK_TARGET`, `BULK_LIMIT`).

In times of high demand or high competition, we'd allow the price to more closely reflect that within one period. In addition, by requiring current tenants to renew at a (close-to) market price, we allow to translate that competitive power to them. We still guarantee to pay (if they can pay the market price) but avoids core captures of not-so-useful-projects that just made it in earlier and benefit from the static increase in prices.